dotCMS/src/main/java/com/dotcms/rest/api/v1/categories/CategoriesResource.java

```diff
@@ -271,6 +271,87 @@ public final CategoryView save(@Context final HttpServletRequest  httpRequest,
         }
     }
 
+    /**
+     * Update a working version of an existing category for sortOrder. The categoryEditDTO must contain the inode and sortOrder of the category.
+     *
+     * @param httpRequest       {@link HttpServletRequest}
+     * @param httpResponse      {@link HttpServletResponse}
+     * @param categoryEditDTO  {@link CategoryForm}
+     * @return CategoryView
+     * @throws DotDataException
+     * @throws DotSecurityException
+     */
+    @PUT
+    @Path("/_sort")
+    @JSONP
+    @NoCache
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    public final Response save(@Context final HttpServletRequest  httpRequest,
+            @Context final HttpServletResponse httpResponse,
+            final CategoryEditDTO categoryEditDTO
+            ) throws DotDataException, DotSecurityException {
+
+        final InitDataObject initData = new WebResource.InitBuilder(webResource)
+                .requestAndResponse(httpRequest, httpResponse).rejectWhenNoUser(true).init();
+        final User user = initData.getUser();
+        final Host host = this.categoryHelper.getHost(categoryEditDTO.getSiteId(),
+                () -> this.hostWebAPI.getCurrentHostNoThrow(httpRequest));
+        final PageMode pageMode = PageMode.get(httpRequest);
+
+        Logger.debug(this,
+                () -> "Saving category sortOrder. Request payload is : " + getObjectToJsonString(
+                        categoryEditDTO));
+
+        DotPreconditions.checkArgument(UtilMethods.isSet(categoryEditDTO.getCategoryData()),
+                "The body must send a collection of category inode and sortOrder");
+
+        Category parentCategory = null;
+
+        if (UtilMethods.isSet(categoryEditDTO.getParentInode())) {
+            parentCategory = this.categoryAPI.find(categoryEditDTO.getParentInode(), user,
+                    pageMode.respectAnonPerms);
+        }
+
+        Iterator iterator = categoryEditDTO.getCategoryData().entrySet().iterator();
+
+        while(iterator.hasNext()) {
+            Map.Entry entry = (Map.Entry) iterator.next();
+            String key = (String) entry.getKey();
+            Integer value = (Integer) entry.getValue();
+
+                final Category category = this.categoryAPI.find(key, user, pageMode.respectAnonPerms);
+
+                if (null == category) {
+                    Logger.error(this,
```

Comment:
probably wan't to throw a bad request exception

Reply:
addressed in the last commit